doc/elements.md

```diff
@@ -78,6 +78,29 @@ Attribute | Type | Default | Description
 `digits` | integer | 2 | number of digits that must be correct for `comparison="sigfig"` or `comparison="decdig"`.
 `eps_digits` | integer | 3 | Additional digits (beyond `digits`) used to compute a grace tolerance.
 
+## `pl_file_upload` element
+
+```html
+<pl_file_upload files="['foo.py','bar.c']" />
+```
+
+Provides a way to accept file uploads as part of an answer. They will be stored
+in [the format expected by externally graded questions](externalGrading.md#file-submission-format).
+
+Attribute | Type | Default | description
+--- | --- | --- | ---
+`answers_name` | string | \_file | Variable name to store data in. **For externally graded questions, you should rely on the default.**
+`filenames` | JSON array | [] | List of files that should and must be submitted.
```

Comment:
So, I wasn't really sure what to call it either... I was trying to avoid having to do a bunch of escaping for quotes, which looks pretty ugly. I actually end up inlining this as a variable assignment in javascript. I guess that breaks if we ever want to parse the file list as JSON in python. Do you think requiring escaping is fine?